_vendor/src/github.com/gorilla/context/doc.go

```diff
@@ -5,6 +5,12 @@
 /*
 Package context stores values shared during a request lifetime.
 
+Note: gorilla/context, having been born well before `context.Context` existed,
```

Comment:
see in https://github.com/pingcap/pd/blob/master/glide.yaml#L97, it updates to the latest version automatically.